common/src/main/java/com/wynntils/core/config/ConfigHolder.java

```diff
@@ -32,7 +32,7 @@ public class ConfigHolder {
     private boolean userEdited = false;
 
     public ConfigHolder(
-            Configurable parent, Field field, FeatureCategory category, Config metadata, Type typeOverride) {
+            AbstractConfigurable parent, Field field, FeatureCategory category, Config metadata, Type typeOverride) {
```

Comment:
The point in having a separation of interface and abstract base class is that in all cases except when extending the abstract base class, the functionality should be referred to using the interface.